src/migrations/selenium-javascript.js

```diff
@@ -16,7 +16,7 @@ class SeleniumJavaScriptMigration extends SDKMigration {
   }
 
   transforms = [{
-    message: 'SDK exports have changed, update imports?',
+    message: 'The SDKs package name changed, update imports?',
```

Comment:
Either `The SDK package name...` or `The SDK's package name`

Reply:
no